keyboards/handwired/not_so_minidox/keymaps/default/keymap.c

```diff
@@ -0,0 +1,127 @@
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+    LCTL,    A,    S,    D,    F,    G,             H,    J,    K,    L, SCLN, QUOT,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+    LSFT,    Z,    X,    C,    V,    B,             N,    M, COMM,  DOT, SLSH,  ENT,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+                        LGUI, LOWR,  SPC,       SPC, RASE, RALT
+                    // `----+-----+-----`     `----+-----+-----`
+   ),
+
+  [_LOWER] = LAYOUT(
+// ,----+-----+-----+-----+-----+-----.         ,----+-----+-----+-----+-----+-----,
+     ESC,    1,    2,    3,    4,    5,             6,    7,    8,    9,    0,  DEL,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+    ____, ____, ____, ____, ____, ____,          ____, ____, ____, LCBR, RCBR, BSLS,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+    ____, ____, ____, ____, ____, ____,          ____, ____, ____, ____, ____, ____,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+                        ____, ____, ____,      ____, ____, ____
+                    // `----+-----+-----`     `----+-----+-----`
+  ),
+
+  [_RAISE] = LAYOUT(
+ // ,----+-----+-----+-----+-----+-----.         ,----+-----+-----+-----+-----+-----,
+      GRV, EXLM,   AT, HASH,  DLR, PERC,          CIRC, AMPR, ASTR, LPRN, RPRN,  DEL,
+ // |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+     ____, ____, ____, ____, MINS,  EQL,          ____,   UP, ____, LBRC, RBRC, PIPE,
+ // |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+     ____, ____, ____, ____, ____, ____,          LEFT, DOWN, RGHT, ____, ____, ____,
+ // |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+                         ____, ____, ____,      ____, ____, ____
+                      // `----+-----+-----`     `----+-----+-----`
+   ),
+  [_ADJUST] = LAYOUT(
+// ,----+-----+-----+-----+-----+-----.         ,----+-----+-----+-----+-----+-----,
+    ____, ____, ____, ____,  RST, ____,          ____, ____, ____, ____, ____,  CAD,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+    ____, ____, ____, ____, ____, ____,          ____, ____, ____, ____, ____, ____,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+    ____, ____, ____, ____, ____, ____,          ____, ____, ____, ____, ____, ____,
+// |----+-----+-----+-----+-----+-----|         |----+-----+-----+-----+-----+-----|
+                        ____, ____, ____,      ____, ____, ____
+                    // `----+-----+-----`     `----+-----+-----`
+  )
+};
+
+void persistant_default_layer_set(uint16_t default_layer) {
```

Comment:
This function is depreciated. 

Instead, you can (and probably should) use `set_single_persistent_default_layer(layer)`, as per:
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/keymap.c#L190

This has the advantage of looking cleaner, but also supporting audio by default (if the board has it enabled)
https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/keymaps/default/config.h#L10